lib/engine/game/g_1817_wo.rb

```diff
@@ -30,6 +31,15 @@ def self.title
       def setup
         super
         @new_zealand_city = hexes.find { |hex| hex.location_name == 'Nieuw Zeeland' }.tile.cities[0]
+        # Put an 1867 style green token in the New Zealand hex
+        @green_token = Token.new(nil, price: 0, logo: '/logos/1817/nz.svg', type: :neutral)
+        @new_zealand_city.exchange_token(@green_token)
+      end
+
+      def event_nieuw_zeeland_available!
+        # Remove the 1867-style green token from the New Zealand hex
+        @log << 'Corporations can now be IPOed in Nieuw Zeeland'
+        @green_token.amp(&:remove!)
```

Comment:
what is amp? you mean map?

did you test this code?

Reply:
shouldn't it just be @green_token.remove!

Reply:
I tested then I cleaned up, which introduced the typo. I'm going to add a spec for 1817WO and change it to @green_token.remove!`